geode-core/build.gradle

```diff
@@ -111,10 +112,8 @@ dependencies {
     exclude module: 'aopalliance'
     exclude module: 'asm'
     exclude module: 'cglib'
-    exclude module: 'guava'
     exclude module: 'spring-aop'
-    exclude module: 'spring-context-support'
-    ext.optional = true
+  ext.optional = true
```

Comment:
Also a nit-pick, but `spotless` doesn't examine the build files.  This needs to tab'd in.